xarray/conventions.py

```diff
@@ -353,6 +354,43 @@ def _update_bounds_attributes(variables):
                 if 'calendar' in attrs:
                     bounds_attrs.setdefault('calendar', attrs['calendar'])
 
+def _update_bounds_encoding(variables):
+    """Adds time encoding to time bounds variables.
+
+    Variables handling time bounds ("Cell boundaries" in the CF
+    conventions) do not necessarily carry the necessary attributes to be
+    decoded. This copies the encoding from the time variable to the
+    associated bounds variable so that we write CF-compliant files.
+
+    See Also:
+
+    http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/
+         cf-conventions.html#cell-boundaries
+
+    https://github.com/pydata/xarray/issues/2565
+    """
+
+    # For all time variables with bounds
+    for v in variables.values():
+        attrs = v.attrs
```

Comment:
A general question -- would we consider `'bounds'` to be an encoding parameter (like `'units'` or `'calendar'`)?  In other words should we expect it to be in the `encoding` dictionary or `attrs` dictionary at this stage?  I feel like it may be more intuitive as part of `encoding`, but currently I know that we don't treat it that way when decoding files.

Reply:
In my mental model, encoding attributes are those that control on-disk representation of the data. `bounds` counts as an attr to my mind since it's an attribute that links the variable to another variable. 

A definition or list of what goes in `encoding` and what goes in `attrs` would make a good addition to the docs.

Reply:
> In my mental model, encoding attributes are those that control on-disk representation of the data.

I think this is fair; I guess I was going off of the mental model of encoding parameters defined as "attributes that are potentially required for decoding all the variables in a file," in which case `'bounds'` could qualify.  I think your definition is probably cleaner, because it requires that encoding parameters control how the variable they are attached to is represented on disk (as opposed to another variable).